cpp/src/parquet/column_reader.h

```diff
@@ -364,10 +373,15 @@ class PARQUET_EXPORT RecordReader {
   }
 
   /// \brief Decoded values, including nulls, if any
+  /// FLBA and ByteArray types do not use this array and read into their own
+  /// builders.
   uint8_t* values() const { return values_->mutable_data(); }
 
-  /// \brief Number of values written including nulls (if any)
-  /// There is no read-ahead/buffering for values.
+  /// \brief Number of values written, including space left for nulls if any.
+  /// If this Reader was constructed with read_dense_for_nullable(), there is no space for
```

Comment:
I didn't see any function named `read_dense_for_nullable()`. Should we use `read_dense_for_nullable_` instead?

Reply:
Done.